README.md

```diff
@@ -1,18 +1,151 @@
-# Deploy HPCC Systems Platform with Terraform
-
-This set of Terraform examples deploys all the available features that come with the HPCC Systems OSS Terraform modules.
-
-## Order of deployment
-| Order | Name      | Required |
-| ----- | --------- | :------: |
-| 1     | `VNet`    |   yes    |
-| 2     | `AKS`     |   yes    |
-| 3     | `Storage` |    no    |
-| 4     | `Logging` |    no    |
-| 5     | `AKS`     |    no    |
-| 6     | `HPCC`    |   yes    |
-
-## Modules
-|                Name                 | Source | Used in |
-| :---------------------------------: | :----: | :-----: |
-| `terraform-azurerm-virtual-network` |        | `VNet`  |
\ No newline at end of file
+# Deploy HPCC Systems on Azure under Kubernetes
```

Comment:
I just noticed that this project creates log files in ~/tflogs, a directory that is created if it does not exist.  This directory needs to be documented so the user knows to look there for troubleshooting reasons or perhaps to clean things up after executing this code.

Bonus points for adding a mechanism that avoids that logging (or creating the log directory) as a runtime option.

Reply:
Please address previous comment.

Reply:
Fixed in commit, 4d1b78f